staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -351,48 +384,138 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
 		return err
 	}
 
-	// operationPollRateLimiter.Accept() is a no-op if rate limits are configured off.
```

Comment:
all the clients should support both read and write rate limits

Reply:
I figured that with per client rate limiting we don't need to segregate READ and WRITE ops.

Reply:
we still need because read and write throttling values from Azure ARM API and RPs